roles/1-prep/tasks/raspberry_pi.yml

```diff
@@ -67,15 +67,15 @@
     name: dphys-swapfile
     state: restarted
 
-- name: Install RPi rootfs resizing (iiab-rpi-max-rootfs.sh) and its systemd service (iiab-rpi-root-resize.service), from templates
+- name: Install RPi rootfs resizing (/usr/sbin/iiab-rpi-max-rootfs.sh) and its systemd service (/etc/systemd/system/iiab-rpi-root-resize.service), from templates (root:root by default)
   template:
     src: "{{ item.src }}"
     dest: "{{ item.dest }}"
-    owner: root
-    group: root
+    #owner: root
+    #group: root
     mode: "{{ item.mode }}"
   with_items:
-    - { src: 'iiab-rpi-max-rootfs.sh', dest: '/usr/sbin/iiab-rpi-max-rootfs.sh', mode: '0755'}
+    - { src: 'iiab-rpi-max-rootfs.sh', dest: '/usr/sbin/', mode: '0755'}
```

Comment:
FWIW dest: using the full path is required when the template ends with .j2 (to denote softcoding) so it's generally preferred to keep using the full path anyway. Not sure why the full path was not good enough and now does not match the stanza directly below.

Reply:
When .j2 templating is not in use, there is the DRY principle (https://en.wikipedia.org/wiki/Don%27t_repeat_yourself) to consider, i.e. one can avoid specifying the exact same filename twice in a row

Reply:
At least be consistent apply that process to the line below also....

Reply:
Done:

- PR #2884